kythe/go/extractors/config/smoke/smoke.go

```diff
@@ -111,8 +113,10 @@ type Result struct {
 	FileCoverage float64
 }
 
-func (g Harness) TestRepo(ctx context.Context, repo string) (Result, error) {
-	fromRepo, err := g.filenamesFromRepo(ctx, repo)
+// TestRepo compares the result of extraction and optional indexing with
+// expected output from a given repo.
+func (h Harness) TestRepo(ctx context.Context, repo string) (Result, error) {
```

Comment:
I didn't notice this the last time, but it's better to use the same type for all the methods, unless you have a specific reason to do otherwise (here, some are on the pointer, some on the base type—either is probably fine, but it would be good to be consistent).

Reply:
Ah yea I mean to consolidate on not-pointer since it's not being mutated and doesn't need it.  (Or at least that's my vague understanding)